mhlib/app.go

```diff
@@ -15,15 +15,21 @@
 package mhlib
 
 import (
+	"bytes"
+	"errors"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"regexp"
+	"text/template"
 
 	"github.com/imdario/mergo"
 	"github.com/sirupsen/logrus"
 
 	"github.com/codeskyblue/go-sh"
+	"github.com/ghodss/yaml"
+	"github.com/hairyhenderson/gomplate"
```

Comment:
Did we work out how to use gomplate as a lib? Did it require something upstream to merge?

Reply:
Upstream already merged.